### PR TITLE
Be explicit about business hours

### PR DESCRIPTION
**Pull request by btrd**
Ping @aurelien-reeves-scalingo

### CODE REVIEW

**Comment by yanjost** [comment]
Why the change ?

**Comment by btrd** [comment]
It wasn't clear what our business hours were

See https://scalingo.slack.com/archives/C01U5K16FM2/p1668436076225989?thread_ts=1668434669.778539&cid=C01U5K16FM2

File: src/_posts/security/procedures/2000-01-01-external-testing.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ You can run a load test / smoke test / endurance test on your application under
 
 - Ask the support using support [at] scalingo.com or use the support chat
 - You should communicate the metrics of your test to the support (number of connections...)
-- The test should only be run during business hours
+- The test should only be run during business hours (9AM to 12AM - 2PM to 6PM)
```
Comment (aurelien-reeves-scalingo) [inline src/_posts/security/procedures/2000-01-01-external-testing.md:16]: I would indicate the timezone.
Or UTC times

Comment (btrd) [inline src/_posts/security/procedures/2000-01-01-external-testing.md:16]:
```suggestion
- The test should only be run during business hours - 08:00 to 11:00 and 13:00 to 15:00 (UTC)
```

File: src/_posts/security/procedures/2000-01-01-external-testing.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ You can run a load test / smoke test / endurance test on your application under
 
 - Ask the support using support [at] scalingo.com or use the support chat
 - You should communicate the metrics of your test to the support (number of connections...)
-- The test should only be run during business hours
+- The test should only be run during business hours - 08:00 to 11:00 and 13:00 to 15:00 (UTC)
```
Comment (yanjost) [inline src/_posts/security/procedures/2000-01-01-external-testing.md:16]:
```suggestion
- The test should only be run during business hours - 08:00 to 11:00 and 13:00 to 15:00 (UTC+1, Europe/Paris timezone)
```

**Review by yanjost — CHANGES_REQUESTED** [review]

**Review by aurelien-reeves-scalingo — CHANGES_REQUESTED** [review]

File: src/_posts/security/procedures/2000-01-01-external-testing.md [hunk 3]
```diff
@@ -47,4 +47,4 @@ For example:
 4. If there are any items in the report that would be of interest to Scalingo,
    please send us the information so that we can act accordingly
 
-{% include security_testing_warning.md %}
\ No newline at end of file
+{% include security_testing_warning.md %}
```
Comment (btrd) [inline src/_posts/security/procedures/2000-01-01-external-testing.md:50]:
```suggestion
```

Comment (btrd) [inline src/_posts/security/procedures/2000-01-01-external-testing.md:50]: It's really not clear on Github interface, but this is adding a end of file new line

**Pull request merged**